dashboard/config/application.rb

```diff
@@ -100,8 +100,6 @@ class Application < Rails::Application
       jquery.handsontable.full.css
       jquery.handsontable.full.js
     )
-    config.react.variant = :development
```

Comment:
Until we get to that point, it probably makes sense to build just the prod version? Seems a shame to lose warnings on staging though, since I've definitely found things that way before.